src/rt/dwarfeh.d

```diff
@@ -765,3 +774,10 @@ enum LsdaResult
     cleanup,    // cleanup found (i.e. finally or destructor)
     handler,    // handler found (i.e. a catch)
 }
+
+void terminate(uint line) @nogc
+{
+    static char* p;
+    printf("dwarfeh(%u) fatal error\n", line);
+    *p = 0;     // generate seg fault
```

Comment:
Actually, the seg fault doesn't even work here, because etc.linux.memoryerror installs a signal handler for seg faults which, you guessed it, converts it into a thrown exception and then we get the recursive crash again. So I'm switching it to `abort()`.